src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html

```diff
@@ -0,0 +1,49 @@
+
+<div class="container">
+
+    <h1>YOUR AVAILABILITY</h1>
+
+    <table>
+        <thead>
+            <tr>
+                <th></th>
+                <th>MONDAY</th>
+                <th>TUESDAY</th>
+                <th>WEDNESDAY</th>
+                <th>THURSDAY</th>
+                <th>FRIDAY</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr ng-repeat="slotsForDay in slotsForWeek">
+                <td>{{slotTimes[$index]}}</td>
+                <td ng-repeat="slot in slotsForDay track by $index" ng-class="{'available-slot': slot.available}">
+                    <input type="checkbox" ng-model="slot.available">
+                    <p ng-show="slot.available">AVAILABLE</p>
+                    <p ng-hide="slot.available">BUSY</p>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+    <button type="button" ng-click="updateSlots()">SUBMIT SLOTS</button>
+
+    <style>
```

Comment:
This style should be moved to a separate file - one team already added a main.css file, move it when the file is merged